apps/website/components/SectionGrid/SectionGrid.tsx

```diff
@@ -0,0 +1,32 @@
+import { Section } from "@/components";
+
+interface SectionGridProps {
+  title: string;
+  gridConfig: {
+    headers: Array<string>;
+    examples: Array<() => JSX.Element>;
+  };
+}
+
+export const SectionGrid = ({ title, gridConfig }: SectionGridProps) => (
```

Comment:
I think this is too complex. I would just render a title and a child component and leave the rendering logic to  the child component. 

Also, IMO there is no need for this kind of abstraction since we are figuring out how to display stuff on the ui page. Copy and paste works pretty well 😄 

But we can have something that styles the title.

Reply:
having a simple section abstraction is good to separate section, but I'm with Leo, something more simple would be better as we don't know how we want to display it yet.

Reply:
Got it! Reverting these for now